appveyor.yml

```diff
@@ -1,12 +1,12 @@
 version: Beet-{build}
 pull_requests:
   do_not_increment_build_number: true
-image: Visual Studio 2017
+image: Visual Studio 2022
 max_jobs: 3
 platform:
   - x64
 environment:
-  nodejs_version: 12
+  nodejs_version: 16
   GITHUB_TOKEN:
       secure: jr08/oNuLLnJAth0P0/KslT6bJPfGGA/QkkPWo83z82i0R+U6jJMZxWVrqDBMXKL
 cache:
```

Comment:
I guess the permission issue Appveyor reports (`EPERM: operation not permitted, unlink`) can be solved by not caching the node modules?
https://github.com/phoenixframework/phoenix/issues/2464 -> https://github.com/okonet/lint-staged/issues/274 -> https://github.com/okonet/lint-staged/pull/289

Reply:
Doing so made some progress, now failing to minify bitsharesjs (develop branch):

```
$ cross-env NODE_ENV=production minify build/bitsharesjs.js --mangle --outFile build/bitsharesjs.min.js
node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module './_baseGetTag'
Require stack:
```

Reply:
`yarn run dist` works in my beet dev environment, so it's something environmental in appveyor